### ISSUE TITLE
preprocess for test data

### ISSUE THREAD

**Issue by brotherniuniu** [MIC-DKFZ/nnUNet#1407]
Hello!
The dataset fingerprint and preprocessed .npz and .npy files were provided on the training data. I would like to know whether the same steps are required for the test set data during model testing and the corresponding preprocessed data file will be generated.
Looking forward to your reply, thank you!

**Comment by FabianIsensee** [comment]
Hey, for the test set none of that will be created. The test images will be preprocessed and immediately predicted without storing the preprocessed images. The reason this is done for the training set is because the preprocessed images must be accessed over and over again during the entire training
